sdks/java/core/src/main/java/org/apache/beam/sdk/coders/AtomicCoder.java

```diff
@@ -35,11 +35,18 @@
   /**
    * {@inheritDoc}.
    *
-   * @throws NonDeterministicException
+   * <p>By default, does not throw.
+   *
+   * @throws NonDeterministicException if the coder is not deterministic
    */
   @Override
   public void verifyDeterministic() throws NonDeterministicException {}
 
+  /**
+   * {@inheritDoc}.
+   *
+   * @return null
```

Comment:
It's a good question.

We should probably remove the `@Nullable` annotation and the ability to return null from this method, and then should be able to port away from the returning of nulls at a later point.